carbonmark/pages/api/certificates/[beneficiaryAddress]/[retirementIndex]/index.ts

```diff
@@ -45,7 +45,9 @@ export default async function handler(
     if (!retirement) {
       return res.status(404).send("Not found");
     }
-
+    if (retirement.retire.credit.project.registry !== "ICR") {
```

Comment:
Is this something we should be cleaning up at the api? If the number of decimals for the amount depends on the registry we should be standardising this.